server/users/users.model.ts

```diff
@@ -2,41 +2,31 @@ import { DomainModel } from "../core/domain.model"
 import ValidationException from "../core/exceptions/validation.exception"
 import { UserSportSubscriptionsModel } from "../users/user-sport-subscriptions.model"
 import { UserSettingsModel } from "../users/user-settings.model"
-import { ensureLongerThan } from "../core/validation"
 
 export class UserModel implements DomainModel {
 	public readonly id?: number
 	public readonly email: string
-	public readonly password: string;
 	public readonly isAdmin: boolean
 	public readonly subscriptions: UserSportSubscriptionsModel[]
 	public readonly settings: UserSettingsModel
 	
-	public readonly totalSubscriptions: number
-
 	constructor(
 		id: number | undefined,
 		email: string,
-		password: string,
 		isAdmin: boolean,
 		subscriptions: UserSportSubscriptionsModel[],
 		settings: UserSettingsModel
 	) {
 		this.id = id
 		this.email = email
-		this.password = password
```

Comment:
@jurebb I removed this as I think it's safer we keep it out of domain model. Plus it's not really used right now, apart from length validation which is done in service anyway.